### DISCUSSION TITLE
求个数据导出功能

### DISCUSSION THREAD

**Discussion by zeoxion** [Planshit/Tai#43] [Ideas]
如果大佬能加个导出csv文件的功能就好了。感谢感谢！

**Comment by noberumotto** [comment]
稍晚些版本一定会有的，目前很多基础功能需要完善

**Comment by noberumotto** [comment]
@zeoxion 抱歉有些晚了，数据导出功能已经加入了，如果还在用Tai的话记得更新~

**Reply by zeoxion** [reply]
芜湖! 谢谢大佬~

**Reply by Wenbobobo** [reply]
> @zeoxion 抱歉有些晚了，数据导出功能已经加入了，如果还在用Tai的话记得更新~

蟹蟹大佬😘
刚刚试了发现网页的数据好像导出不了，是需要额外操作嘛？
在软件里是正常显示的，如果也能支持导出备份就好了

**Comment by MC-dusk** [comment]
但是只能导出设置了分类的数据也太迷了吧，一个分类没设就只能导出空表。。导出的时候自动默认分类不就好了。网页导出也没有
